test/expected-output-test/expressions/print.expected

```diff
@@ -0,0 +1,20 @@
+a(k int).
+
+b(k int,
+  p text,
+  q text,
+  r int).
+
+c(s text,
+  n int,
+  t text).
+
+Q( "test"  :: TEXT, 123, id, unnest(y)) * :-
+    a(id),
+    b(id, x, y, z),
+    c(x || y, 10,  "foo" ), [z > 100]; [[![z < 20]], [z < 50]].
```

Comment:
The most outer `[ ]` is not necessary as `[P; Q]` is equivalent to `P; Q`. But if it's clearer, I think it makes sense to wrap the whole condition using a `[ ]`.